src/topology/subset_properties.lean

```diff
@@ -789,15 +803,63 @@ end
 variables (α) [sigma_compact_space α]
 open sigma_compact_space
 
-/-- An arbitrary compact covering of a σ-compact space. -/
+/-- A monotone compact covering of a σ-compact space. -/
 def compact_covering : ℕ → set α :=
-classical.some exists_compact_covering
+accumulate exists_compact_covering.some
 
 lemma is_compact_compact_covering (n : ℕ) : is_compact (compact_covering α n) :=
-(classical.some_spec sigma_compact_space.exists_compact_covering).1 n
+compact_accumulate (classical.some_spec sigma_compact_space.exists_compact_covering).1 n
 
 lemma Union_compact_covering : (⋃ n, compact_covering α n) = univ :=
-(classical.some_spec sigma_compact_space.exists_compact_covering).2
+begin
+  rw [compact_covering, Union_accumulate],
+  exact (classical.some_spec sigma_compact_space.exists_compact_covering).2
+end
+
+@[mono] lemma compact_covering_subset ⦃m n : ℕ⦄ (h : m ≤ n) :
+  compact_covering α m ⊆ compact_covering α n :=
```

Comment:
```suggestion
@[mono] lemma monotone_compact_covering : monotone (compact_covering α) :=
```

Reply:
This way `have := monotone_compact_covering α h` will output `≤`, not `⊆`.

Reply:
I see, thanks for explaining.